helpers/chromebook-tests

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+try_then_test() {
```

Comment:
the way we currently start/run our test is using this /init script (which we add in an initramfs):

https://git.linaro.org/ci/job/configs.git/tree/lt-qcom-linux-integration/builders.sh#n44

I have been willing to move that over here in bootrr , and come up with a nice/clean/generic init script instead of our hack above. 

I think the 'for loop' we do based on DT compatible is a nice way to have a generic implementation. Perhaps better than doing try_then_test for each platform, no?

Reply:
Nice, I was looking and wondering myself how you launched the tests but I didn't find. Thanks for the link.

Absolutely agree, I'll remove this patch for now and send a pull request with the other two. I'll look at your init script too. Thanks.